model_analyzer/triton/server/server_handler.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from model_analyzer.triton.server.server_config import TritonServerConfig
+from model_analyzer.triton.server.server_factory import TritonServerFactory
+
+from model_analyzer.config.input.config_utils import binary_path_validator
+
+from model_analyzer.constants import CONFIG_PARSER_FAILURE, LOGGER_NAME
+
+from urllib.parse import urlparse
+
+import os
+import logging
+
+logger = logging.getLogger(LOGGER_NAME)
+
+
+class TritonServerHandler():
+    """
+    Static methods used to create and return Triton Servers
```

Comment:
Is there much of a difference between the factory and the functionality of this class? Just thinking out loud and wondering if they should be combined.

Reply:
I've removed the handler and moved all the code into the factory